wallet/src/de/schildbach/wallet/service/platform/PlatformSyncService.kt

```diff
@@ -194,7 +194,7 @@ class PlatformSynchronizationService @Inject constructor(
     override suspend fun updateContactRequests() {
 
         // if there is no wallet or identity, then skip the remaining steps of the update
-        if (platformRepo.hasIdentity || walletApplication.wallet == null) {
+        if (!platformRepo.hasIdentity || walletApplication.wallet == null) {
```

Comment:
These are some bugs leftover from merging Core 19 changes, masternode changes, etc -- but we didn't actually test this to see if it was working.